shoes-core/lib/shoes/common/style.rb

```diff
@@ -139,6 +140,11 @@ def set_visibility(new_styles)
         update_visibility
       end
 
+      def set_coloring(new_styles)
+        update_fill   if new_styles.include?(:fill)
+        update_stroke if new_styles.include?(:stroke)
+      end
```

Comment:
Love how the common style module makes stuff like this so damn easy - thanks @KCErb  :)